### PR TITLE
Added Support for onChange when not in view

### PR DESCRIPTION
**Pull request by suyashvash**
Earlier it was working fine if isVisible was true and doing the desired actions but if the component goes out of the view area (isVisible == false) it wasn’t triggering the action ( not returning the false value in onChange). So I added another condition that it will still send the isVisible boolean in props onChange even if it is not in the view area. Useful when you want to pause a video when not in user view.

Now you can trigger any action if not in area

### CODE REVIEW

**Comment by suyashvash** [comment]
Dude I just did it in my app

**Comment by SvanBoxel** [comment]
Thank you for this PR and fix! :octocat:

**Comment by SvanBoxel** [comment]
Thanks for this PR! Could you check the linter output? It seems something is broken. 
Screenshot 2022-09-13 at 14 52 58

File: src/index.tsx [hunk 1]
```diff
@@ -70,7 +70,9 @@ const VisibilitySensor: React.FC<Props> = (props) => {
     if (lastValue !== isVisible) {
       setLastValue(isVisible);
       props.onChange(isVisible);
-    }
+    }else{
+      props.onChange(isVisible);
+     }
```
Comment (SvanBoxel) [inline src/index.tsx:75]: I think this should fix the linter error

```suggestion
    } else {
      props.onChange(isVisible);
    }
```

**Comment by suyashvash** [comment]
Yup, it did!

**Comment by SvanBoxel** [comment]
✨  Nice work! 

:octocat:

**Pull request merged**